.buildkite/longruns/pipeline.yml

```diff
@@ -289,13 +282,35 @@ steps:
 
       - label: "GPU AMIP FINE: new target amip: topo + diagedmf"
         key: "gpu_amip_target_topo_diagedmf"
-        command: "srun julia --color=yes --project=experiments/AMIP/ experiments/AMIP/coupler_driver.jl --config_file $CONFIG_PATH/gpu_amip_target_topo_diagedmf.yml"
+        command:
+          - echo "--- Run simulation"
+          - "srun julia --color=yes --project=experiments/AMIP/ experiments/AMIP/coupler_driver.jl --config_file $CONFIG_PATH/gpu_amip_target_topo_diagedmf.yml"
         artifact_paths: "experiments/AMIP/output/amip/gpu_amip_target_topo_diagedmf_artifacts/*"
+        timeout_in_minutes: 1440
+        env:
+          CLIMACOMMS_CONTEXT: "MPI"
         agents:
-          slurm_gpus: 1
-          slurm_mem: 16GB
+          queue: clima
+          slurm_ntasks: 4
+          slurm_gpus: 4
+          slurm_mem: 20GB
+          modules: common
         soft_fail: true
 
+  # DYAMOND AMIP: 1 day (convection resolving) - running on clima, so need to instantiate again
```

Comment:
I think we can remove the "running on clima, so need to instantiate again" part of this comment